doc/sections/commands.md

```diff
@@ -207,6 +207,20 @@ arguments on the left side of the `≔` symbol (similarly to a value declaration
 in OCaml). Some arguments can be declared as implicit by enclosing them in
 curly brackets.
 
+<!---------------------------------------------------------------------------->
+### `inductive`
+The `inductive` command is used to define an inductive type, as in Coq.
```

Comment:
No need to speak of Coq here ;-)

Thanks to explain what is the effect of the command.

Reply:
Done